detections/endpoint/ssa___detect_dump_lsass_memory_using_comsvcs.yml

```diff
@@ -14,7 +14,7 @@ search: '| from read_ssa_enriched_events() | eval tenant=ucast(map_get(input_eve
   "string", null), process_name=lower(ucast(map_get(input_event, "process_name"),
   "string", null)), timestamp=parse_long(ucast(map_get(input_event, "_time"), "string",
   null)), process=lower(ucast(map_get(input_event, "process"), "string", null)), event_id=ucast(map_get(input_event,
-  "event_id"), "string", null) | where process_name LIKE "%rundll32.exe%" AND match_regex(process,
+  "event_id"), "string", null) | where cmd_line IS NOT NULL AND process_name LIKE "%rundll32.exe%" AND match_regex(process,
```

Comment:
Do we need to be null-checking `cmd_line` in this detection, or should we be checking for nulls in `process_name` before we use a LIKE function on it?

Reply:
we prob should let me update the detection

Reply:
fixed